docs/integrations/destinations/databricks.md

```diff
@@ -271,11 +271,11 @@ Please keep these data directories on S3/Azure. Otherwise, the corresponding tab
 
 Each table will have the following columns:
 
-| Column                             |   Type    | Notes                                                          |
-| :--------------------------------- | :-------: | :------------------------------------------------------------- |
-| `_airbyte_ab_id`                   |  string   | UUID.                                                          |
-| `_airbyte_emitted_at`              | timestamp | Data emission timestamp.                                       |
-| Data fields from the source stream |  various  | All fields in the staging files will be expanded in the table. |
+| Column                             |   Type    | Notes                    |
+| :--------------------------------- | :-------: | :----------------------- |
+| `_airbyte_ab_id`                   |  string   | UUID.                    |
+| `_airbyte_emitted_at`              | timestamp | Data emission timestamp. |
+| Data fields from the source stream |  various  |                          |
```

Comment:
Any reason we don't call out this column as `_airbyte_data` with an entry in the note column that it's "Data fields from the source stream as JSON object" or something like this? Data field**s** sounds like it would be multiple columns

Reply:
Agree!